core/src/main/java/net/opentsdb/query/interpolation/types/numeric/NumericInterpolator.java

```diff
@@ -176,6 +176,17 @@ public TimeStamp nextReal() {
     return fill_policy;
   }
 
+  @Override
+  public void close() {
+    try {
+      iterator.close();
+    } catch (IOException e) {
+      // Don't bother logging.
+      e.printStackTrace();
```

Comment:
may be remove the `printStackTrace` too.

Reply:
I do want to keep these around just to see if they're every executed. They shouldn't be but it's possible at some point in the future.